### ISSUE TITLE
Create dummy image with text within it

### ISSUE THREAD

**Issue by mosbth** [mosbth/cimage#101]
Sometimes when one create webpages one needs a placeholder for a image yet to come. A dummy image with some text within it. 

Would be nice if img.php could create such a dummy image to use for placeholder.

**Comment by mosbth** [comment]
Partly implemented in v0.7.6. Creates dummy images but without text.

**Comment by mosbth** [comment]
Added to the config file.

```
     /**
      * The name representing a dummy image which is automatically created
      * and stored at the defined path.
      * The dummy_dir must be writable and it defaults to a subdir of the
      * cache directory.
      * Write protect the dummy_dir to prevent creation of new dummy images,
      * but continue to use the existing ones.
      *
      * Default value:
      *  dummy_enabled:  true as default, disable dummy feature by setting
      *                  to false.
      *  dummy_filename: 'dummy' use this as ?src=dummy to create a dummy image.
      *  dummy_dir:      Defaults to subdirectory of 'cache_path',
      *                  named the same as 'dummy_filename'
      */
      //'dummy_enabled' => true,
      //'dummy_filename' => 'dummy',
      //'dummy_dir' => 'some writable directory',
```

Review the testfile `webroot/test/test_issue101-dummy.php` for how it works.

If problems, check that the cache-directory is writable, `cache/dummy`.
